NTO/ServiceManagement/entities/ChangeRequest.ttl

```diff
@@ -61,6 +61,7 @@ ogit.ServiceManagement:ChangeRequest
 		
 	);
 	ogit:allowed (
+			[ ogit:belongs  ogit:Person]
```

Comment:
again if this is "who created it", it would be Person --creates-->ChangeRequest

Reply:
After another thought, we can use the `[ ogit:requests  ogit.ServiceManagement:ChangeRequest ]` from person

Reply:
good point :) then please remove this :)